test/e2e/petset.go

```diff
@@ -581,14 +721,22 @@ func (p *petSetTester) setHealthy(ps *apps.PetSet) {
 			framework.Failf("Found multiple non-healthy pets: %v and %v", pod.Name, markedHealthyPod)
 		}
 		p, err := framework.UpdatePodWithRetries(p.c, pod.Namespace, pod.Name, func(up *api.Pod) {
-			up.Annotations[petset.PetSetInitAnnotation] = "true"
+			up.Annotations[petset.PetSetInitAnnotation] = value
 		})
 		ExpectNoError(err)
 		framework.Logf("Set annotation %v to %v on pod %v", petset.PetSetInitAnnotation, p.Annotations[petset.PetSetInitAnnotation], pod.Name)
 		markedHealthyPod = pod.Name
 	}
 }
 
+func (p *petSetTester) setHealthy(ps *apps.PetSet) {
```

Comment:
this is too easy and we know it works from all the other tests. Basically any test that invokes `saturate` does exactly this. 

Can you actually force a pet to become unhealthy? i.e it needs to fail a readiness probe or crashloop. The easiest way to do this is probably run nginx sidecar (or just as the pet) serving a file (as the readiness probe endpoint). remove the file with `kubectl exec` and it serves some non-200 code, put the file back and it's happy.